src/__tests__/BoxAnnotations-test.js

```diff
@@ -21,49 +21,61 @@ describe('BoxAnnotations', () => {
         beforeEach(() => {
             permissions = {
                 can_annotate: false,
+                can_create_annotations: false,
+                can_view_annotations: false,
                 can_view_annotations_all: false,
                 can_view_annotations_self: false,
             };
         });
 
-        it('should return false if permissions do not exist', () => {
+        test('should return false if permissions do not exist', () => {
```

Comment:
Why not use the alias?

Reply:
The convention is `it` = Karma and `test` = Jest. Most of the other tests use `test`.